app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -532,4 +534,20 @@ public void onFailure(Call<ResponseBody> call, Throwable t) {
                 });
     }
 
+    /**
+     * Sets maximum message content editing limit in seconds
+     *
+     * @param seconds seconds to be set as maximum limit
+     */
+    private void setMessageContentEditLimitSeconds(int seconds) {
+        SharedPreferences preferences = getSettings();
+
+        //Checks id maximum content edit limit is already saved or not
+        if (!preferences.getBoolean(Constants.IS_MAXIMUM_CONTENT_EDIT_LIMIT_SAVED, false)) {
```

Comment:
@timabbott Can `realm_message_content_edit_limit_seconds` change over time for a particular realm configuration? I couldn't find a way to edit it in the dev environment.